smartsim/entity/dbnode.py

```diff
@@ -45,31 +47,53 @@ class DBNode(SmartSimEntity):
     into the smartsimdb.conf.
     """
 
-    def __init__(self, name, path, run_settings, ports):
+    def __init__(self, name, path, run_settings, ports, hostname_converter=None):
```

Comment:
``host_map`` maybe? I like to go for nouns for structures and verbs for methods/functions